### DISCUSSION TITLE
Blanket web clone

### DISCUSSION THREAD

**Discussion by terenc3** [rafaelmardojai/blanket#236] [Show and tell]
I've create a clone of Blanket for the web

https://apps.roanapur.de/blanket/

Feel free to add feedback or suggestions.

![Screenshot](https://codeberg.org/terenc3/blanket-web/raw/branch/master/screenshot.png?v3)

FYI i've moved the code to https://codeberg.org/terenc3/blanket-web

**Comment by rafaelmardojai** [comment]
Thanks for sharing! Looks really cool.

Maybe I would mimic more Blanket and make the volume range always draggable. But really nice work!

**Comment by bleov** [comment]
Here's my version: https://pikarocks.dev/blanket/
